docs/client.rst

```diff
@@ -72,18 +73,118 @@ WBEM Operation recording
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
 The WBEM client library API provides the possibility to record the WBEM
-operations that are executed on a connection. This is disabled by default
-and can be enabled by setting the
-:attr:`~pywbem.WBEMConnection.operation_recorder` instance variable of the
-:class:`~pywbem.WBEMConnection` object to an operation recorder object,
-i.e. to an object of a subclass of :class:`~pywbem.BaseOperationRecorder`.
-
-Typical usage scenarios for operation recorders are the tracing of WBEM
-operations, or the generation of test cases.
-
+operations that are executed on a connection.
 WBEM Operation recording uses the classes and subclasses defined in
 :ref:`WBEM Operation Recorder`.
 
+This is disabled by default and can be enabled by adding recorders to a
+WBEMConnection with the method
+:meth:`~pywbem.WBEMConnection.add_operation_recorder`.
+
+Typical usage scenarios for various operation recorders are the tracing of WBEM
+operations, tracing of interactions with a WBEM server, or the generation of
+test cases.
+
+Please not that the method of activating operation recorders changed starting
+with pywbem 0.11.0 and the addition of a second recorder.  See
+:meth:`~pywbem.WBEMConnection.add_operation_recorder` for more information.
+
+This adds the recorder defined in the method call to a list of active recorders
+in WBEMConnection. All active recorders are called for each WBEMConnection
+method that sends information to the WBEM server.
+
+A single recorder can be  be disabled with
+:meth:`~pywbem.BaseOperationRecorder.disable` method and enabled with
+:meth:`~pywbem.BaseOperationRecorder.enable` method.
+
+The logger names for the operations and http loggers must be
+defined using  the :class:`~pywbem:`PywbemLoggers` class to
```

Comment:
Remove the middle back quote in the link above.

Reply:
Fixed